spark/dl/src/main/scala/com/intel/analytics/bigdl/example/dlframes/imageTransferLearning/README.md

```diff
@@ -0,0 +1,86 @@
+## Overview
+   Deep Learning Frames provides high-level APIs for scalable deep learning in Scala with Apache Spark.
+   The current version of Deep Learning Frames provides a suite of tools around working with and processing images using deep learning. 
+   this exmaple demostrates how to use BigDL for transfer learning.
+
+## Transfer Learning 
+   1. DLFrames provides utilities to perform transfer learning on images, which is one of the fastest (code and run-time-wise) ways to start using deep learning.             
+         
+          val createLabel = udf((name: String) => if (name.contains("cat")) 1.0 else 0.0)
```

Comment:
We already got the example code, so I'm thinking either remove code here or only leave the most critical lines.

Reply:
will delete a couple of lines